celer/PN_logreg.pyx

```diff
@@ -377,9 +378,9 @@ cpdef int PN_logreg(
                 notin_WS, X_mean, weights_pen, center, 0)
 
         for i in range(n_samples):
-            aux[i] /= max(1, norm_Xaux)
+            aux[i] /= max(1, norm_Xaux / alpha)
```

Comment:
The rescaling here is not implemented like in the other files.
Yet, I assume that we should divide by ``alpha``.

Reply:
yes, makes sense
do you identify why the build fails?